lib/active_model/serializer/adapter/json.rb

```diff
@@ -7,7 +7,7 @@ class Json < Base
 
         def serializable_hash(options = nil)
           options ||= {}
-          { root => Attributes.new(serializer).serializable_hash(options) }
+          { root => Attributes.new(serializer, instance_options).serializable_hash(options) }
```

Comment:
@bf4 thanks for the explanation -- now if only comments about that were in the code :-)